doc/classes/Panel.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="Panel" inherits="Control" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		Provides an opaque background for [Control] children.
+		A GUI control that provides a [StyleBox] background for its child controls.
```

Comment:
This is a bit misleading tbh. Panel is just a panel, it doesn't need any child controls nor any "foreground" controls.

Reply:
> A GUI control that displays a [StyleBox].

Is this better?

Reply:
Yes.